tensorflow/lite/micro/tools/generate_cc_arrays.py

```diff
@@ -70,16 +70,26 @@ def generate_array(input_fname):
     for byte in image_bytes:
       out_string += hex(byte) + ","
     return [len(image_bytes), out_string]
+  elif input_fname.endswith('.wav'):
+    wav = wave.open(input_fname, mode='r')
```

Comment:
super nit: wav --> wav_file or something more descriptive. I letter difference between module name and variable name makes me nervous.

Reply:
Good call. Done.